plaso/parsers/sqlite_plugins/chrome_history.py

```diff
@@ -15,8 +15,14 @@ class ChromeHistoryFileDownloadedEventData(events.EventData):
   """Chrome History file downloaded event data.
 
   Attributes:
+    danger_type (int): enum indicating if the download was dangerous.
+    end_time (int): when the download ended.
     full_path (str): full path where the file was downloaded to.
+    interrupt_reason (int): enum indicating why the download stopped.
+    opened (int): if the downloaded file was opened from the browser.
     received_bytes (int): number of bytes received while downloading.
+    start_time (int): when the download started.
```

Comment:
Why not make this a separate event? Also what does the int represent?

Reply:
start_time is currently used as the event time; I thought with the additional of end_time it was more clear to have start_time as an attribute as well, even if it's duplicative. I expanded the description.